.database.rc

```diff
@@ -0,0 +1,2404 @@
+# Ganga configuration file ($Name: Ganga-8-3-1 $). DO NOT remove this line.
```

Comment:
I do not think this file should be included. The configuration file is a private file for the user that is created on startup if not there, and not a file form the repo.